ios/UtilsPlugin.mm

```diff
@@ -33,7 +33,9 @@ - (void) performActionForShortcutItem:(UIApplicationShortcutItem *) shortcutItem
 }
 
 - (void) updateShortcutItems: (NSDictionary *)jsonObject {
-    NSUserDefaults *sharedDefaults = [[NSUserDefaults alloc] initWithSuiteName:@"group.com.hashcube.sudokuquest"];
+    NSString *appGroup = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"WidgetGroup"];
```

Comment:
groups are needed to share data between apps. so, if you want to share data between a widget and an app, you need both to be in same group